integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonCreateTableCommand.scala

```diff
@@ -78,8 +78,8 @@ case class CarbonCreateTableCommand(
         path
       }
       val streaming = tableInfo.getFactTable.getTableProperties.get("streaming")
-      if (path.startsWith("s3") && streaming != null && streaming != null &&
-          streaming.equalsIgnoreCase("true")) {
+      if (streaming != null &&
+        streaming.equalsIgnoreCase("true") && path.startsWith("s3")) {
```

Comment:
"true”.equalsIgnoreCase(streaming) will judge the code “streaming != null” automatically

Reply:
Statement "tableInfo.getFactTable.getTableProperties.get("streaming")" can return null if there is no property called streaming, so i think we shall have this check.

Reply:
better to keep  "if" condition in one line